structuralcodes/core/_section_results.py

```diff
@@ -175,37 +175,86 @@ class UltimateBendingMomentResults:
 
 
 @dataclass
-class NMMInteractionDomain:
+class InteractionDomain:
+    """Class for storing common data on all interaction domain results.
+
+    Data contained:
+
+    strains: a numpy array with shape (n, 3) containing ea, ky and kz.
+    forces: a numpy array with shape (n, 3) containing n, my and mz.
+    field_num: a numpy array with shape (n,) cotaning a number between
+    1 and 6 indicating the failure field.
```

Comment:
Very good that you documented this. Please consider slightly rewriting to the google format, i.e.:

```
...
    Attributes:
        strains (numpy.Array): A numpy array ...
        forces (numpy.Array): A numpy array ...
...
```

Reply:
done in 6b1ed715f89c7ad4f5f0772b8d48e73501484073

Reply:
Thanks, only a minor typo in the field num which should be between 1 and 6👍🏻

Reply:
sorry! Done!